tools/assetpipelineeditor/exportdialog.cpp

```diff
@@ -0,0 +1,88 @@
+#include "exportdialog.h"
+#include "ui_exportdialog.h"
+
+#include <private/gltf2exporter_p.h>
+
+#include <QFile>
+#include <QFileInfo>
+#include <QDir>
+#include <QJsonDocument>
+#include <QFileDialog>
+#include <QStyle>
+#include <QPushButton>
+#include <QMessageBox>
+
+ExportDialog::ExportDialog(
+        Kuesa::GLTF2Exporter &exporter,
+        const QString &path,
+        QWidget *parent)
+    : QDialog(parent), ui(new Ui::ExportDialog), m_exporter(exporter), m_originalFile(path), m_targetFile(m_originalFile)
```

Comment:
OTOH it'd be best to have one per line here.

Reply:
done